app/gather/assets/css/_survey-view.scss

```diff
@@ -351,16 +405,24 @@
    }
   }
 
-  .survey-data-toolbar {
-    padding: 1rem 5vw;
-    height: $toolbar-height-mobile;
+  .survey-data-tabs {
+
+    .tab {
+      padding: 1rem 0 1rem;
```

Comment:
Lint warnings

```text
./css/_survey-view.scss
  411:16  warning  Property `padding` should be written more concisely as `1rem 0` instead of `1rem 0 1rem`  shorthand-values
```